src/components/NavBar/OidcLoginComponent.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import { TextField, Box, Button } from '@mui/material';
+import { LoginButton } from '@inrupt/solid-ui-react';
+
+// Utility Imports
+import { SOLID_IDENTITY_PROVIDER } from '../../utils';
+
+// Custom Hook Imports
+import { useRedirectUrl } from '../../hooks';
+
+const OidcLoginComponent = () => {
+  const [oidcIssuer, setOidcIssuer] = useState(SOLID_IDENTITY_PROVIDER);
+  const redirectUrl = useRedirectUrl();
+
+  return (
+    <>
+      <Box sx={{ flexGrow: 1 }} />
+      <Box sx={{ flexGrow: 1 }} />
```

Comment:
I believe these Boxes are here to help keep the components for TextField and the LoginButton on the right.

I think 1 Box should be enough since they're already using flexGrow: 1. The first box should fill out the space.